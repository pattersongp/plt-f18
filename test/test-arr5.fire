func int main = () => {
        
        array[int, str] a;
        array[int, int] n;
        a[0] = "GATE";
        n[0] = 0;
        a[1] = "GATE";
        n[1] = 1;
        a[2] = "PARASAMGATE";
        n[2] = 2;
        a[3] = "BODHI SAVHA";
        n[3] = 3;


        int j = 0;
        while(j < 4) {
        	sprint(a[j]);
        	print(n[j]);
        	j = j + 1;
        }

        return 0;
}