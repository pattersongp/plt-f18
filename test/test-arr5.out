GATE
0
GATE
1
PARASAMGATE
2
BODHI SAVHA
3